docs/general/builtins/scalar.rst

```diff
@@ -824,6 +824,12 @@ synopsis::
 the time will have second precision, no fractional seconds (microseconds)
 are given.
 
+.. NOTE::
+   No guarantee is provided about the accuracy of the underlying clock.
+   Applications requiring a more accurate clock must implement abstract
+   class ``java.time.Clock`` themselves using a different external clock,
+   such as an NTP server.
+
```

Comment:
This is user facing documentation and there is no way a user is going to implement a custom `Clock` class. 

I meant that we need to point out that depending on the system the accuracy of the results may be limited to milliseconds.